src/go/log/log_test.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
```

Comment:
This is another best practice maybe? that I don't see in our internal (so far) `Golang Best Practices` authored by @hcgatewood

Reply:
I think there's currently use of `mockery`, but that lib is more relaxed on expectations and type checking:

https://blog.codecentric.de/2019/07/gomock-vs-testify/#expectations

Thus, I lean towards gomock, but am open to sticking with testify/mockery if we want

Reply:
Leaving discussion to you and @hcgatewood  =D